packages/graph/hash_graph/bin/hash_graph/src/args.rs

```diff
@@ -37,17 +57,24 @@ pub struct Args {
     #[clap(
         long,
         default_value_t = Regex::new(r"http://localhost:3000/@(?P<shortname>[\w-]+)/types/(?P<kind>(?:data-type)|(?:property-type)|(?:entity-type)|(?:link-type))/[\w\-_%]+/").unwrap(),
-        env = "HASH_GRAPH_ALLOWED_URL_DOMAIN_PATTERN"
+        env = "HASH_GRAPH_ALLOWED_URL_DOMAIN_PATTERN",
+        global = true
     )]
     pub allowed_url_domain: Regex,
 
     /// The OpenTelemetry protocol endpoint for sending traces.
-    #[clap(long, default_value = None, env = "HASH_GRAPH_OTLP_ENDPOINT")]
+    #[clap(long, default_value = None, env = "HASH_GRAPH_OTLP_ENDPOINT", global = true)]
     pub otlp_endpoint: Option<String>,
 
     /// Generate a completion script for the given shell and outputs it to stdout.
     #[clap(long, value_enum, exclusive = true)]
     generate_completion: Option<Shell>,
```

Comment:
Maybe we want to make this a subcommand as well?

Reply:
Good idea! Will do :)

Reply:
Great, thanks! I have a few more comments on this, but will capture these in separated threads.